content/terraform/v1.12.x/docs/language/block/stack/deploy/deployment.mdx

```diff
@@ -0,0 +1,294 @@
+---
+page_title: deployment block reference for Terraform Stack deployments
+description: Use the `deployment` block to define how many times you want to deploy your Stack's infrastructure.
+---
+
+# `deployment` block reference
+
+Use the `deployment` block to define how many times you want to deploy your Stack's infrastructure. You can define `deployment` blocks in a `<NAME>.tfdeploy.hcl` file, which is the deployment configuration file for your Stack.
+
+## Background
+
+<Note>
+
+HCP Terraform supports up to a maximum of 20 deployments per Stack.
+
+</Note>
+
+The `deployment` block defines how to deploy your Stack's infrastructure. Each deployment represents a separate instance of your Stack infrastructure. Each deployment runs in its own isolated agent, wholly separated from other Stack deployments.
+
+Each Stack requires at least one `deployment` block, and you can add additional `deployment` blocks every time you want to deploy your Stack's infrastructure again with different input values. To learn more, refer to [Define deployment configuration](/terraform/language/stacks/deploy/config).
+
+You can define the following configuration blocks in a `tfdeploy.hcl` file:
+- Use the [`deployment_group`](/terraform/language/block/stack/deploy/deployment-auto-group) block to create a group of deployments and apply rules to those deployments.
+- Use the [`deployment_auto_approve`](/terraform/language/block/stack/deploy/deployment-auto-group) block to define rules that automatically approve deployment plans based on specific conditions.
```

Comment:
```suggestion
- Use the [`deployment_auto_approve`](/terraform/language/block/stack/deploy/deployment-auto-group) block to define rules that automatically approve deployment plans based on specific conditions.
- Use the [`deployment_group`](/terraform/language/block/stack/deploy/deployment-auto-group) block to assign `deployment_auto_approve` rules to specific deployments.
```

Until we get the cooler group stuff running, I think that's an easier way for a user to sort this out in their head? It does mean this description needs an update once we do build all that, but it seems likely to need that anyway.